### ISSUE TITLE
The convergence problem of albert xlarge finetuning

### ISSUE THREAD

**Issue by itsucks** [google-research/google-research#151]
I'm finetuning albert xlarge model with my joint training task of intent classification and slot filling,  and I tried many times, the model never converge. 

Then I repalced the albert model with bert base, the code is working.

What's wrong? SOS

**Comment by jinmel** [comment]
seems like xlarge config requires larger batch size on default. Mine didn't converge as well.

**Comment by itsucks** [comment]
> seems like xlarge config requires larger batch size on default. Mine didn't converge as well.

I set learning rate to 5e-6, it worked.
